facet-core/src/types/field.rs

```diff
@@ -54,6 +54,10 @@ pub struct FieldBuilder {
 pub enum FieldAttribute {
     /// Marks field as containing sensitive information
     Sensitive,
+    /// Skip serializing this field.
+    SkipSerializing,
+    /// Skip serializing this field if the function identified by `.0` evaluates to true.
+    SkipSerializingIf(&'static str),
```

Comment:
Wouldn't it make more sense for these to be encoded in `Arbitrary` somehow, having `facet-json` (and other serializers) parse that our of arbitrary? Otherwise this now ties serialization behavior into facet-core.

Reply:
I just followed the logic of `#[facet(rename = "...")]` (`Rename` down below), which does the exact same thing

Reply:
Well rename is a more general concept, not specific to serialization.

Reply:
I’m open to suggestions, but I think it would be hard to do arbitrary attributes that are actually symbols? We can try and figure out some syntax I suppose